lib/chef-dk/helpers.rb

```diff
@@ -113,14 +129,33 @@ def git_windows_bin_dir
       @git_windows_bin_dir ||= File.expand_path(File.join(omnibus_root, "embedded", "git", "usr", "bin"))
     end
 
+    def habitat_embedded_bin_dir
+      @habitat_embedded_bin_dir ||= ENV["HAB_WS_EMBEDDED_BIN_DIR"]
+    end
+
+    def habitat_bin_dir
+      @habitat_bin_dir ||= ENV["HAB_WS_BIN_DIR"]
+    end
+
     #
-    # environment vars for omnibus
-    #
+    # provides sane environment variables for running
+    # Workstation and DK tools.
     def omnibus_env
       @omnibus_env ||=
         begin
           user_bin_dir = File.expand_path(File.join(Gem.user_dir, "bin"))
-          path = [ omnibus_bin_dir, user_bin_dir, omnibus_embedded_bin_dir, ENV["PATH"] ]
+          path = []
+
+          # TODO - cases where the omnibus install exists alongside the hab install
```

Comment:
This is probably no different than having a gem install + an omnibus install in terms of getting a reliable result out of `omnibus_env`